### PR TITLE
Testing: Add visual regression testing with Chromatic

### LINKED ISSUE #3363: [Enhancement] Visual regression tests in Chromatic
- [x] Set up Chromatic
- [x] Run Chromatic via cli and publish from local machine
- [x] Move Chromatic project ownership from @jkaltoft to a shared profile or similar (and confirm it still works)

### LINKED ISSUE #3364: [Enhancement] Github Action for Chromatic
- [x] Add github action to run visual tests on PRs: https://www.chromatic.com/docs/github-actions/

### PR DESCRIPTION
**Pull request by jkaltoft**
## Which issue does this PR close?

This PR closes #3363
This PR closes #3364 

## Does this PR introduce a breaking change?

- [ ] Yes
- [x] No

## Checklist:

The following tasks should be carried out in sequence in order to follow [the process of contributing](https://github.com/kirbydesign/designsystem/blob/develop/.github/CONTRIBUTING.md#the-process-of-contributing) correctly.

### Reminders
- [ ] Make sure you have implemented tests following the guidelines in: "[The good: Test](https://github.com/kirbydesign/designsystem/wiki/The-Good%3A-Test)".
- [ ] Make sure you have updated the cookbook with examples and showcases (for bug fixes, enhancements & new components).

### Review  
- [ ] Determine if your changes are a fix, feature or breaking-change, and add the matching label to your PR. If it is tooling, dependency updates or similar, add ignore-for-release.
- [ ] Do a [self-review](https://github.com/kirbydesign/designsystem/wiki/The-Good%3A-Self-review).
- [x] Request that the changes are code-reviewed 
- [ ] Request that the changes are [UX reviewed](https://github.com/kirbydesign/designsystem/blob/develop/.github/CONTRIBUTING.md#ux-review) (only necessary if your PR introduces visual changes)

When the pull request has been approved it will be merged to `develop` by Team Kirby.

### CODE REVIEW

File: .github/workflows/chromatic.yml [hunk 1]
```diff
@@ -0,0 +1,23 @@
+name: "Chromatic"
+
+on:
+    pull_request:
+        types: [ labeled ]
```
Comment (RasmusKjeldgaard) [inline .github/workflows/chromatic.yml]: If we add the synchronize label here, we can run the chromatic action whenever a new commit is added, and the label is present. That might be a better default?
```suggestion
        types: [labeled, synchronize]
```

File: .github/workflows/chromatic.yml [hunk 2]
```diff
@@ -0,0 +1,23 @@
+name: "Chromatic"
```
Comment (RasmusKjeldgaard) [inline .github/workflows/chromatic.yml]: Consider a more descriptive name 
```suggestion
name: "Visual Regression Tests - Chromatic"
```

File: .github/workflows/chromatic.yml [hunk 3]
```diff
@@ -0,0 +1,23 @@
...
+
+on:
+    pull_request:
+        types: [ labeled ]
+
+jobs:
+  chromatic:
+    if: ${{ github.event.label.name == 'chromatic' }}
```
Comment (RasmusKjeldgaard) [inline .github/workflows/chromatic.yml]: If we add the synchronize pull request type, we also need to check for the label name on the pull-request itself.

```suggestion
    if: ${{ github.event.label.name == 'chromatic' || contains(github.event.pull_request.labels.*.name, 'chromatic') }}
```

**Review by RasmusKjeldgaard — CHANGES_REQUESTED** [review]

File: .github/workflows/chromatic.yml (hunk 1 above)
Comment (jkaltoft) [reply .github/workflows/chromatic.yml:5]: This (and the condition check on the job below) is exactly what I was looking for. Thanks.

**Review by jkaltoft — COMMENTED** [review]
The suggested changes are exactly what I was looking for. Thanks.

File: .github/workflows/chromatic.yml (hunk 2 above)
Comment (jkaltoft) [reply .github/workflows/chromatic.yml]: I agree that this should be a descriptive name. I just went with the simplest name to get started 🙂

**Review by RasmusKjeldgaard — DISMISSED** [review]

**Review by RasmusKjeldgaard — DISMISSED** [review]

**Review by RasmusKjeldgaard — APPROVED** [review]
🎉

**Pull request merged**